dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -33,6 +61,26 @@ function Sounds() {
   this.soundsById = {};
 }
 
+/**
+ * @param {Array.<string>} filenames ending in (.mp3|.ogg|.wav)
```

Comment:
Could perhaps use a little more explanation. I assume this function is used to register multiple filenames for a single sound id? Should it also make sure we don't try to repeat any extensions (i.e. check that soundRegistrationConfig[extension] is undefined before we assign to it.